modules/hydra-master-common.nix

```diff
@@ -4,7 +4,7 @@ with lib; {
   nix = {
     # autoOptimiseStore = true;
     extraOptions = ''
-      allowed-uris = https://github.com/NixOS https://github.com/input-output-hk
+      allowed-uris = https://github.com/NixOS https://github.com/input-output-hk https://github.com/moretea
```

Comment:
Instead of adding moretea to allowed URIs, can use of that repo be turned into a pure nix fetch?

Reply:
I'm not sure what kind of fetch is pure enough... this one is from `niv` using `fetchFromGitHub` presumably.